tests/beam_design/test_beam_design.py

```diff
@@ -9,20 +9,31 @@ def test_beam_design():
     https://www.structuralguide.com/worked-example-singly-reinforced-beam-design-using-ec2/
     """
     b,d = bd.section_dimension_rule_of_thumb(span=6750) #mm
-    MaxMoment,MaxShearForce =  bd.max_bending_moment_and_shear_force(span=6750,load=36e3,load_type="point_load")
+    MaxMoment,MaxShearForce =  bd.max_bending_moment_and_shear_force(span=6750,point_load=36e3,distributed_load=0)
     out=  bd.beam_section_design(span =6750,
                               b = b,
                               d = d,
                               MaxMoment=MaxMoment,
                               MaxShearForce=MaxShearForce,
                               fck=20,
                               fyk=500,
-                              steelDia=16,
+                              steelDia=12,
                               cover=25)
     for key,val in out.items():
         print(f"{key} --> {val}")
-    assert(out["bottom_steel_numbers"]==2)
+    assert(out["bottom_steel_numbers"]==3)
```

Comment:
Maybe I'm looking in the wrong file, but in test beam design only bottom_steel_numbers is tested, the ratio of required area of steel to given is not tested, which I guess we might use for the optimization (since I thought we wanted to prescribe the number of steel bars and then only having the other concrete parameters to be flexible in the design.

Reply:
You are looking at an outdated file.
[This](https://github.com/BAMresearch/LebeDigital/blob/66-beam-design/tests/beam_design/test_beam_design.py) is the current version, which only tests the difference of area as the final result, due to the reasons you have explained ;)